corehq/apps/styleguide/templates/styleguide/bootstrap5/atoms/accessibility.html

```diff
@@ -108,50 +101,139 @@ <h5>A11y check</h5>
+      <li>Keep in mind that this list is not exhaustive and covers only a portion of what makes a site accessible to
+        all users. Ensuring accessibility is an ongoing commitment to creating inclusive digital experiences for
+        everyone.</li>
   </ul>
-  <h2 id="screen-readers" class="pt-4">
-    Designing for Screen Readers
+
+  <h2 id="focusability" class="pt-4">
+    Focusability, Tabability, and Visibility
   </h2>
-  <p>
-    In 2020, USH began a web accessibility improvement effort, specifically surrounding screen readers.
-    <a href="https://docs.google.com/document/d/1fHNaVlaFl1xrbtPbMqj0YpByss1e0_hzCRlAUbVLboM/edit?usp=sharing">This document</a> details
-    an overview of the effort, deficiencies found by an outside agency of CommCare, recommendations, and resources.
-  </p>
-  <p>
-    Quick ways to test for web accessibility*
-  </p>
-  <ul>
-    <li>
-      Keyboard Navigation: Testing if modified code is navigable by keyboard is an effective way to
-      test for accessibility. Many users rely on keyboard navigation that are otherwise unable to use a mouse and it’s
-      also a good indicator if it will be visible to a screen reader. Read more about potential barriers to effective
-      keyboard navigation and keyboard testing <a href="https://webaim.org/techniques/keyboard/">here</a>.
-    </li>
-    <li>
-      Explore the accessibility tree: An accessibility tree is a part of your website's DOM. All
-      accessible objects are created in the accessibility tree for each DOM element that needs to be exposed to
-      assistive technologies. From it, you can see how the page elements are included and organized. To view the
-      accessibility tree of a page under development, we can use
-      <a href="https://developers.google.com/web/tools/chrome-devtools/accessibility/reference">Chrome DevTools' accessibility feature</a>.
-    </li>
-    <li>
-      <em>*this is by no means a complete list and covers only a portion of what makes a site accessible to all users</em>
-    </li>
-  </ul>
+  <h5>tabindex Attribute</h5>
+    <p>The tabindex attribute specifies the tab order of an element and determines whether an element can receive
+      keyboard focus. Here are some scenarios where you might use tabindex:</p>
+    <ul>
+        <li>Focusable but Not Natively Focusable Elements: Some elements like &lt;div&gt; or &lt;span&gt; are not
+          natively focusable, but you might want them to receive focus for user interaction. In such cases, you can
+          add tabindex="0" to make them focusable. Note: we avoid using custom focus order using tab index with values
+          above 0. Instead, make sure DOM elements are ordered correctly for navigability, then modify visual
+          positioning with CSS if needed.</li>
+        <li>Remove Focusability: to explicitly remove an item</li>
+        <li>Dynamic Focus Control: If you have dynamically generated or hidden elements that need to become focusable
```

Comment:
Curious, do you know if we do this? I'm realizing I assumed that making an element invisible would also remove it from tabbing, is that not the case?